src/components/functional/ledger/components/ledger-confirm/ledger-confirm.tsx

```diff
@@ -20,7 +20,11 @@ export class LedgerConfirm {
 
   render() {
     const ledgerConfirmationItems: LedgerConfirmationItem[] = [
-      { label: this.confirmScreenData.confirmAddressText, value: this.confirmScreenData.selectedAddress, explorerLink: this.confirmScreenData.explorerLink },
+      {
```

Comment:
LedgerConfirmationItem --> `LedgerConfirmationItemType` ?
and maybe rename into something more clear? LedgerConfirmDataType ?